cli/testdata/bpmetadata/md/list-content.md

```diff
@@ -29,3 +29,7 @@ It creates projects and configures aspects like Shared VPC connectivity, IAM acc
 
 ### preDeploy
 To deploy this blueprint you must have an active billing account and billing permissions.
+
+### Architecture
```

Comment:
From a UX perspective, embedding the diagram here would let users correlate the diagram and the text.

Reply:
Yeah I thought about that. Again requires more changes but we should do it. I just wanted this initial fix to go out so we can share a better version of metadata with the partners.